extensions/ql-vscode/src/databases/ui/db-selection-decoration-provider.ts

```diff
@@ -13,7 +13,8 @@ export class DbSelectionDecorationProvider implements FileDecorationProvider {
   ): ProviderResult<FileDecoration> {
     if (uri?.query === "selected=true") {
       return {
-        badge: "✔",
+        badge: "●",
```

Comment:
This "badge" has to be a letter or text character, so we can't use the exact same icon as in the command (see [above](https://github.com/github/vscode-codeql/pull/1840/files#r1040927490)).